generator/ServiceClientGeneratorLib/Generators/Endpoints/EndpointResolver.partial.cs

```diff
@@ -14,7 +14,7 @@ private string GetValueSource(Parameter parameter)
         {
             switch (parameter.builtIn)
             {
-                case "AWS::Region": return "config.RegionEndpoint?.SystemName";
+                case "AWS::Region": return "requestContext.Request.AlternateEndpoint?.SystemName ?? config.RegionEndpoint?.SystemName;";
```

Comment:
There's an extra `;` here, I ran the generator and the resolvers ended up with:
```csharp
result.Region = requestContext.Request.AlternateEndpoint?.SystemName ?? config.RegionEndpoint?.SystemName;;
```

Reply:
Removed the extra `;`